client/src/app/domain/models/users/user.constants.ts

```diff
@@ -9,8 +9,22 @@ export const userHeadersAndVerboseNames: { [key in keyof User]?: string } = {
     default_password: `Initial password`,
     email: `Email`,
     username: `Username`,
-    gender: `Gender`,
+    gender: `Gender`
+};
+
+export const accountHeadersAndVerboseNames: { [key in keyof User]?: string } = {
```

Comment:
Please export this constant in a dedicated file for accounts under the account directory